op-plasma/damgr_test.go

```diff
@@ -21,6 +21,49 @@ func RandomData(rng *rand.Rand, size int) []byte {
 	return out
 }
 
+// TestAdvanceL1Origin test advance l1 origin when there are multiple commitments in the same block
+func TestAdvanceL1Origin(t *testing.T) {
+	var (
+		logger        = testlog.Logger(t, log.LevelDebug)
+		ctx           = context.Background()
+		bn     uint64 = 1
+		l1F           = &mockL1Fetcher{}
+		err    error
+	)
+	l1F.ExpectFetchReceipts(common.HexToHash(""), nil, types.Receipts{}, nil)
+	l1F.ExpectL1BlockRefByNumber(bn, eth.L1BlockRef{Number: bn}, nil)
+
+	// simulate one block has multiple commitments
+	state := NewState(logger, &NoopMetrics{})
+	state.SetInputCommitment([]byte("0x1"), bn, 0)
```

Comment:
It would be nice to have a non-zero challenge window here to more closely follow the delayed finalization.